Gems/Atom/Tools/AtomToolsFramework/Code/Source/Application/AtomToolsApplication.cpp

```diff
@@ -334,7 +360,7 @@ namespace AtomToolsFramework
         }
     }
     
-    bool AtomToolsApplication::LaunchDiscoveryService()
+    bool AtomToolsApplication::LaunchLocalServer()
```

Comment:
does this need appsec review?

Reply:
Thank you for bringing this up but I don't believe so. It's already been discussed. This does not actually open a network connection. It uses pipes and the file system for IPC.